module/applications/components/inventory.mjs

```diff
@@ -370,18 +370,19 @@ export default class InventoryElement extends HTMLElement {
 
   /**
    * Handle item actions.
-   * @param {Element} target  Button or context menu entry that triggered this action.
-   * @param {string} action   Action being triggered.
+   * @param {PointerEvent} event  The triggering event.
+   * @param {Element} target      Button or context menu entry that triggered this action.
+   * @param {string} action       Action being triggered.
    * @returns {Promise}
    * @protected
    */
-  async _onAction(target, action) {
-    const event = new CustomEvent("inventory", {
+  async _onAction(event, target, action) {
```

Comment:
This broke the context menus, which are calling `_onAction` but don't have an event to pass on. Perhaps add it into an option object at the end of the list?

Reply:
Oops, good catch. I suppose we *should* have a the event that was fired when the user clicked the context menu option, but it seems core does not provide it in the callback.